services/movie_session.py

```diff
@@ -0,0 +1,47 @@
+import init_django_orm  # noqa: F401
+
+from django.db.models.query import QuerySet
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: str,
+                         movie_id: int, cinema_hall_id: int) -> None:
+    MovieSession.objects.create(show_time=movie_show_time,
+                                movie_id=movie_id,
+                                cinema_hall_id=cinema_hall_id)
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    filters = {}
+
+    if session_date:
+        filters["show_time__date"] = session_date
+
+    return MovieSession.objects.filter(**filters)
+
+
+def get_movie_session_by_id(movie_session_id: int) -> QuerySet:
```

Comment:
It returns instance of MovieSession
```suggestion
def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
```